content/en/docs/troubleshooting/ClusterResourcePlacementEviction.md

```diff
@@ -0,0 +1,260 @@
+
+- If yes, the next step is to check if the `ClusterResourceBinding` object is present for the target cluster or why it 
+was not created and try to create an eviction object once `ClusterResourceBinding` is created.
+- If no, the cluster is not picked by the scheduler and hence no need to retry eviction.
+
+### Multiple CRB is present
+
+Example status with multiple `CRB` objects:
+```
+status:
+  conditions:
+  - lastTransitionTime: "2025-04-17T23:48:08Z"
+    message: Found more than one scheduler decision for placement in cluster targeted
+      by eviction
+    observedGeneration: 1
+    reason: ClusterResourcePlacementEvictionInvalid
+    status: "False"
+    type: Valid
+```
+
+In this case the Eviction object reached a terminal state, it's status has `Valid` condition set to `False`, because
+there is more than one `ClusterResourceBinding` object or Placement present for the `ClusterResourcePlacement` object 
+targeting the member cluster. This is a rare scenario, where one `ClusterResourceBinding` is being deleted while a new 
+`ClusterResourceBinding` is being created.
+
+### PickFixed CRP is targeted by CRP Eviction
+
+Example status for `ClusterResourcePlacementEviction` object targeting a PickFixed `ClusterResourcePlacement` object:
+```
+status:
+  conditions:
+  - lastTransitionTime: "2025-04-21T23:19:06Z"
+    message: Found ClusterResourcePlacement with PickFixed placement type targeted
+      by eviction
+    observedGeneration: 1
+    reason: ClusterResourcePlacementEvictionInvalid
+    status: "False"
+    type: Valid
+```
+
+In this case the Eviction object reached a terminal state, it's status has `Valid` condition set to `False`, because
+the `ClusterResourcePlacement` object is of type `PickFixed`. Users cannot use `ClusterResourcePlacementEviction` 
+objects to evict resources propagated by `ClusterResourcePlacement` objects of type `PickFixed`. The user can instead 
+remove the member cluster name from the `clusterNames` field in the policy of the `ClusterResourcePlacement` object.
+
+## Failed to execute eviction
+
+### Eviction blocked because placement is missing
+
+```
+status:
+  conditions:
+  - lastTransitionTime: "2025-04-22T00:36:44Z"
+    message: Eviction is valid
+    observedGeneration: 1
+    reason: ClusterResourcePlacementEvictionValid
+    status: "True"
+    type: Valid
+  - lastTransitionTime: "2025-04-22T00:36:44Z"
+    message: Eviction is blocked, placement has not propagated resources to target
+      cluster yet
+    observedGeneration: 1
+    reason: ClusterResourcePlacementEvictionNotExecuted
+    status: "False"
+    type: Executed
+```
+
+In this case the Eviction object reached a terminal state, it's status has `Executed` condition set to `False`, because
+for the targeted `ClusterResourcePlacement` the corresponding `ClusterResourceBinding` object's spec is set to 
+`UnScheduled` meaning the scheduler decided to pick a different cluster for the placement.
```

Comment:
Hi Arvind! Just a question here; apologies if I have asked about this before (as I vaguely remember so): for `unscheduled` bindings we blocked eviction, but there might still be running resources on the selected clusters, right? And if the rollout is stuck, these bindings might never get deleted by the rollout controller

Reply:
I guess my concern is that at this moment it might have little to do with the scheduler and the state might not change no matter how long the user has waited.